### ISSUE TITLE
Does table formula support reference to external variables?

### ISSUE THREAD

**Issue by jzhang1** [tgrosinger/md-advanced-tables#84]
I don't know if this is not documented or not implemented. I would like to be able to reference external variables in a table formula. For example, let's say I have a table of rooms with area in sqft, and I would like to add a column of area in sqm. Instead of hard coding a formula like $2=($1 * 10.5), I would like to do something like $2=($1 * !sqft2sqm) where !sqft2sqm is an externally defined variable in the document. So I would be able define the variable else where like in another table and document what that is instead of having magic numbers sprinkled in my formulas. Also, if I use the same constant across multiple formulas I can update them in one place instead of having to remember everywhere I used the same constant.

**Comment by TheGMYeti** [comment]
No, not currently. This could be a duplicate of #40. The only thing I could see this not being a direct duplicate is if the request is more geared toward having general blocks get parsed by the table evaluation as something like a string value that could be manipulated or interpreted later. Regardless, making them both a part of the same system would likely be for the best.

Also, as far as I'm aware, there is also no way to name tables, cell groups, or individual cells, or use table column names as a reference in evaluations.
